src/__tests__/__snapshots__/WidgetView.test.tsx.snap

```diff
@@ -4,6 +4,7 @@ exports[`renders a WidgetView using Snapshots 1`] = `
 <RNWidgetView
   configuration={
     Object {
+      "appearance": 0,
```

Comment:
it's weird to me that the config declaration uses an integer. why can't it use the enum?

Reply:
@stevenou It's a snapshot generated by Jest. We are indeed using an enum, but Jest will convert it to an integer.